.github/renovate.json5

```diff
@@ -3,6 +3,7 @@
     "config:base",
     "github>int128/typescript-action-renovate-config",
   ],
+  "platformAutomerge": true,
```

Comment:
Please don't enable this because this repository does not have a required check in the branch protection rules.
This will merge any pull request even if CI is failed.